docs/pages/plugins-create-plugin.md

```diff
@@ -186,6 +186,14 @@ module.exports = {
 }
 ```
 
+#### Running the plugin
```

Comment:
I'm not sure we want to introduce sketchtool when a user just create a plugin. 99% of the users will just run the plugin via the Sketch Plugin menu

Reply:
Agree with @mathieudutour here, the majority will run the menu through the menu. However there could be a note box:

> *Note:* To run a plugin from the command line you can use [`sketchtool`](/cli)

… or something along the lines